widgets/common/src/RevertToDefaultIcon.jsx

```diff
@@ -8,7 +8,18 @@ function RevertToDefaultIcon({ value, defaultValue, onClick, onMouseDown, popupC
     }
 
     return (
-        <Popup trigger={<Icon name="undo" link onClick={onClick} onMouseDown={onMouseDown} style={style} />}>
+        <Popup
+            trigger={
+                <Icon
+                    aria-label="revert-to-default"
```

Comment:
`aria-label` should be a user-readable and a user-friendly string :slightly_smiling_face: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

It's like a tooltip

Thus, I suggest:

```suggestion
                    aria-label="Revert value to default"
```

Reply:
I had to double-check it :smile: so I guess it's new to both of us